pinot-plugins/pinot-minion-tasks/pinot-minion-builtin-tasks/src/main/java/org/apache/pinot/plugin/minion/tasks/purge/PurgeTaskExecutor.java

```diff
@@ -30,20 +35,29 @@
 import org.apache.pinot.spi.config.table.TableConfig;
 import org.apache.pinot.spi.data.Schema;
 import org.apache.pinot.spi.utils.builder.TableNameBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 
 public class PurgeTaskExecutor extends BaseSingleSegmentConversionExecutor {
   public static final String RECORD_PURGER_KEY = "recordPurger";
   public static final String RECORD_MODIFIER_KEY = "recordModifier";
   public static final String NUM_RECORDS_PURGED_KEY = "numRecordsPurged";
   public static final String NUM_RECORDS_MODIFIED_KEY = "numRecordsModified";
+  private static final ThreadMXBean MX_BEAN = ManagementFactory.getThreadMXBean();
+  private long _startTimeNs;
+  private long _endTimeNs;
+  private long _startAllocatedBytes;
+  private long _endAllocatedBytes;
+  private static final Logger LOGGER = LoggerFactory.getLogger(PurgeTaskExecutor.class);
```

Comment:
nit: move this line to the top so that it can group with other static constants?

Reply:
Addressed comments.